core/src/main/java/hudson/model/Computer.java

```diff
@@ -173,6 +173,8 @@
      */
     protected volatile OfflineCause offlineCause;
 
+    protected volatile OfflineCause temporarilyOfflineCause;
```

Comment:
Can you please add a Javadoc documenting the semantics of the new `temporarilyOfflineCause` field. Additionally the Javadoc of the preceding `offlineCause` field was already vague to begin with, but the existence of the new `temporarilyOfflineCause` field naturally raises the question "how do the semantics differ from the `offlineCause` field?" so please clarify the javadoc of the original `offlineCause` field to explain the difference between the two fields.